api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/MessageProcessorScheduler.java

```diff
@@ -97,4 +97,14 @@ public void processPaymentMessages() {
     public void processReportClaimMessages() {
         messageProcessor.processMessagesOfType(MessageType.REPORT_CLAIM);
     }
+
+    @Scheduled(cron = EVERY_30_SECONDS)
```

Comment:
Shall we turn the offset-by-5 schedule into a constant and use that for all MI reports?

Reply:
What is the "offset-by-5 schedule"?

Reply:
This (for "MessageProcessor:REPORT_CLAIM"):
```
    @Scheduled(cron = "05/35 * * * * *") // every 30 seconds, offset from other schedules by 5 seconds to prevent clashes with CREATE_NEW_CARD for new claims
```

Reply:
Ah yes, let's have a constant for that.

Reply:
Will do in it's own PR, potentially with a custom annotation to group the Schedule annotations together.